fvm/fvm_bench_test.go

```diff
@@ -270,7 +270,7 @@ type txWeights struct {
 	TXHash                string `json:"txHash"`
 	LedgerInteractionUsed uint64 `json:"ledgerInteractionUsed"`
 	ComputationUsed       uint   `json:"computationUsed"`
-	MemoryUsed            uint   `json:"memoryUsed"`
+	MemoryEstimate        uint   `json:"memoryUsed"`
```

Comment:
this relies on the log in `fvm/transactionInvoker.go` so it should now change to `MemoryEstimate        uint   `json:"memoryEstimate"`